test/langtools/tools/javac/classfiles/attributes/innerclasses/InnerClassesInAnonymousClassTest.java

```diff
@@ -58,7 +58,7 @@ public void setProperties() {
     @Override
     public List<TestCase> generateTestCases() {
         currentClassType = ClassType.CLASS;
-        setPrefix("class Anonymous {} {new Anonymous() {");
+        setPrefix("class Anonymous { int f; } {new Anonymous() {");
         List<TestCase> sources = super.generateTestCases();
 
```

Comment:
Hack to suppress ACC_PERMITS_VALUE bit from being set thereby causing the test to "fail"